test/gov/start_mgmt_hub.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+PREFIX="All-in-one management hub deployment:"
+# the environment variables set by the Makefile are:
+#  EXCHANGE_ROOT_PW 
+#  EXCHANGE_IMAGE_TAG
+#  EXCHANGE_DATABASE 
+#  EXCHANGE_HUB_ADMIN_PW
+#  EXCHANGE_SYSTEM_ADMIN_PW
+#  AGBOT_ID
+#  AGBOT_TOKEN
+#  AGBOT_IMAGE_TAG
+#  CSS_IMAGE_TAG
+#  MONGO_IMAGE_TAG
+#  POSTGRES_IMAGE_TAG 
+#  POSTGRES_USER
+#  TEST_VARS
+
+# check if we need start the second agbot
+if [ "$MULTIAGBOT" == "1" ]; then
+    export START_SECOND_AGBOT=true
+else
+    export START_SECOND_AGBOT=false
+fi
+
+echo -e "${PREFIX} START_SECOND_AGBOT setting is ${START_SECOND_AGBOT}."
+
+cd /tmp
+rm deploy-mgmt-hub.sh
```

Comment:
Can we encapsulate this command such that when it fails, we dont get an error message on the screen?

Reply:
done